### PR TITLE
Add bluemix analytics and update dependencies

### PR DESCRIPTION
**Pull request by germanattanasio**

### CODE REVIEW

**Comment by germanattanasio** [comment]
@kognate I'm using an environment variable to inject the HTML we need to enable analytics. That way we don't pollute the demo with code that is not related to the demo

File: server/views/index.ejs [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <link rel="icon" type="image/png" sizes="96x96" href="/images/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
     <link rel="stylesheet" href="/css/main.css">
+    <%- bluemixAnalytics %>
```
Comment (kognate) [inline server/views/index.ejs]: We're stuffing the whole script tagg into the env?

Comment (germanattanasio) [inline server/views/index.ejs]: The other option is to have the script loading for all our users

Comment (germanattanasio) [inline server/views/index.ejs]: `BLUEMIX_ANALYTICS=<script>window._analytics={coremetrics:false,optimizely:false,addRoll:false};</script><meta name="segment" property="watson-demos" value="similarity-search-demo"/><script src="https://console.ng.bluemix.net/analytics/build/bluemix-analytics.min.js"></script>`

**Pull request merged**